### PR TITLE
show notification when the favorited session starts

### LINKED ISSUE #166: Show notification when the favorited session starts
**Idea Description**
The title says everything

**Reference images and links**
You can refer SessionAlarm class.
https://github.com/DroidKaigi/conference-app-2020/blob/master/feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/util/SessionAlarm.kt

### PR DESCRIPTION
**Pull request by torikatsupg**
## Issue
- close #166 

## Overview (Required)
- Show notification when the favorited session starts

## Links
- 

## Screenshot
Before | After
:--: | :--:
| - | 
| - |

### CODE REVIEW

File: app-android/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -3,6 +3,7 @@
     xmlns:tools="http://schemas.android.com/tools">
 
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
```
Comment (takahirom) [inline app-android/src/main/AndroidManifest.xml:6]: 👍

File: app-android/src/main/java/io/github/droidkaigi/confsched2022/notification/NotificationUtil.kt [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+            )
+            setAutoCancel(true)
+            setSmallIcon(io.github.droidkaigi.confsched2022.core.designsystem.R.drawable.ic_app)
+        }
+
+        val notificationManagerCompat = NotificationManagerCompat.from(context)
+        if (ActivityCompat.checkSelfPermission(
+                context,
+                permission.POST_NOTIFICATIONS
+            ) != PackageManager.PERMISSION_GRANTED
```
Comment (takahirom) [inline app-android/src/main/java/io/github/droidkaigi/confsched2022/notification/NotificationUtil.kt:49]: 👍

**Comment by takahirom** [comment]
Your commit is now in https://github.com/DroidKaigi/conference-app-2022/pull/809 🙏

**Comment by torikatsupg** [comment]
@takahirom Understood.
Thanks for addressing the part I left out!

**Pull request closed without merging**